creating-campaign/stripe-payment-feature/stripe.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Stripe Connected Payments'
+---
+
+<Note>To enable the Stripe feature and connect your account to Moongate's Stripe account, please contact Moongate for permission.</Note>
+
+## What is Stripe Connect?
+> Stripe Connect is a service that allows platforms to manage payments on behalf of their users. Here's an example to illustrate how it works:
```

Comment:
The definition is a bit off, and difficult to understand.
Something like: Stripe connect is a feature that would allow your payments to directly go to your source but handle the fees to Moongate
Rephrase this and ask chatGPT to provide a better one.